npm-shrinkwrap.json

```diff
@@ -4233,6 +4238,11 @@
               "from": "isarray@>=1.0.0 <1.1.0",
               "resolved": "https://registry.npmjs.org/isarray/-/isarray-1.0.0.tgz"
             },
+            "isstream": {
```

Comment:
I'm not doubting that this change was needed, but just out of curiosity how was this not here before? Or is it just another side-effect of all the madness of the different NPM versions?